core/src/smartcontracts/wasm.rs

```diff
@@ -848,7 +850,8 @@ impl<'wrld> Runtime<state::Validator<'wrld>> {
         let mut store = self.create_store(state);
         let instance = self.instantiate_module(module, &mut store)?;
 
-        let validate_fn = Self::get_typed_func(&instance, &mut store, export::WASM_MAIN_FN_NAME)?;
+        let validate_fn =
+            Self::get_typed_func(&instance, &mut store, export::VALIDATOR_VALIDATE_FN_NAME)?;
```

Comment:
Because it looks like there is no sence in having it the same as `Smart Conract` and `Trigger` in the same time while we have another entrypoint. To me the new name is more logical

Reply:
Not a big deal, but I liked the uniformity of naming all `entrypoint` functions `_iroha_main`. @Erigara @appetrosyan your vote

Reply:
I vote for preserving old name.

Reply:
I vote for the new naming convention.